lib/types/book.ts

```diff
@@ -0,0 +1,11 @@
+export interface Book {
+  id: string;
+  title: string;
+  description: string;
+  published_date: string;
+  author: {
+    id: string;
+    name: string;
+  };
+  image_url?: string;
```

Comment:
Seeing two different naming conventions here `image_url` and `createdAt` would it be useful to follow 1 pattern?